feature_engineering.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
```

Comment:
- First: native python modules (ie: all modules [here](https://docs.python.org/3/py-modindex.html)) then, newline, then, installed modules as `pandas` etc, then, newline and import custom module as `data_preparation`
- Sort subgroups in alphabetical order modules